test/libsolidity/GasCosts.cpp

```diff
@@ -66,20 +66,21 @@ BOOST_AUTO_TEST_CASE(string_storage)
 			}
 		}
 	)";
+	m_compiler.overwriteReleaseFlag(true);
 	compileAndRun(sourceCode);
 
 	if (Options::get().evmVersion() <= EVMVersion::byzantium())
-		CHECK_GAS(133899, 130591, 100);
```

Comment:
2500 gas for the additional field, even in release mode?

Reply:
It's 9 additional bytes to be stored on chain costing 9*200 = 1800 - that's what our gas estimation tests tell... I'm not sure about the remainder - maybe those 9 bytes have to be sent for the creation transaction costing 9*68 = 612 on top of the 1800? Not sure why that'd be missing from our estimation then, though...